api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/BaseReferenceHandlingTranslator.java

```diff
@@ -43,6 +44,10 @@ protected Reference createEncounterReference(@NotNull OpenmrsObject encounter) {
 		        .setType(FhirConstants.ENCOUNTER);
 	}
 	
+	protected Reference createVisitReference(@NotNull Visit visit) {
+		return new Reference().setReference(FhirConstants.VISIT + "/" + visit.getUuid()).setType(FhirConstants.ENCOUNTER);
```

Comment:
This should still return a valid FHIR reference, so it needs to be `Encounter/<uuid>` rather than `Visit/<uuid>`.

Reply:
Turns out that this wasn't needed anymore, removing it.